.github/workflows/ISSUE_TEMPLATE/not-sure-where-to-raise-an-issue.md

```diff
@@ -0,0 +1,15 @@
+---
+name: ⁉️ Not sure where to raise an issue?
+about: No sure where to raise the issue?
+
+---
+
+Unsure where to raise something? Here are some of the common places:
+
+- If your issue is due to the way OpenJDK is built in a Jenkins environment, then this is the repository to raise it in - Describe your problem below
+- If your issue is related to our end user shell scripts (not in jenkins) raise it in [openjdk-build](https://github.com/AdoptOpenJDK/openjdk-build/)
```

Comment:
More a question than a comment: what does "not in jenkins" mean in this context? Are the openjdk-build scripts no longer the build scripts used during jenkins builds?

Also, though it may seem redundant, I recommend a link to this repo as well, so people know what issues are suitable to raise here.

Reply:
Perhaps "not in jenkins" means "and is reproducible outside of jenkins, e.g. locally"?

Reply:
Adjusted to clarify groovy scripts

Reply:
Much clearer. Good job. :)